%YAML 1.2
---
name: pilot_net_graph
components:
- name: tensor_sub
  type: nvidia::gxf::RosSubscriber
- name: pilotnet_decoder
  type: nvidia::isaac_ros::PilotNetDecoder 
  parameters:
    # Default values, will be overridden by the C++ node
    tensor_name: "output_tensor"
    steer_scale: 1.0
    speed_scale: 1.0
- name: ackermann_pub
  type: nvidia::gxf::RosPublisher
---
# Data flow connections
- source: tensor_sub/output
  target: pilotnet_decoder/input
- source: pilotnet_decoder/output
  target: ackermann_pub/input